proto/config.proto

```diff
@@ -15,9 +15,11 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package proto;
+syntax = "proto2";
+package cockroach.proto;
+option go_package = "proto";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "gogoproto/gogo.proto";
```

Comment:
In python, the (slash-delimited) import path must match the (dot-delimited) package statement in the file. `gogo.proto` is in package `gogoproto`, so we have to import it this way and adjust PROTO_PATH to match. (this becomes moot if we strip out the gogo extensions, but it's still the more proto-standard way to do things)